iml-mailbox/src/lib.rs

```diff
@@ -19,7 +19,9 @@ impl reject::Reject for Errors {}
 pub trait LineStream: Stream<Item = Result<String, warp::Rejection>> {}
 impl<T: Stream<Item = Result<String, warp::Rejection>>> LineStream for T {}
 
-fn streamer(s: BodyStream) -> Pin<Box<dyn Stream<Item = Result<String, warp::Rejection>> + Send>> {
+fn streamer(
+    s: impl Stream<Item = Result<impl Buf, warp::Error>> + Send + 'static,
```

Comment:
Yeah, I also did this locally.

Reply:
Hrrm, I think can remove the static bound... I'll update.